StortSpelProjekt/Engine/src/Renderer/DX12Tasks/CopyOnDemandTask.cpp

```diff
@@ -31,6 +31,56 @@ void CopyOnDemandTask::SubmitTexture(Texture* texture)
 	m_Textures.push_back(texture);
 }
 
+void CopyOnDemandTask::UnSubmitMesh(Mesh* mesh)
+{
+	// Erase uploaded data
+	Resource* uploadResourceVertices = mesh->m_pUploadResourceVertices;
```

Comment:
Should not both of these be "Default" instead of "Upload" or have I misunderstood?

Reply:
It is correct. In this case it doesn't matter. This function is only using the == operator to find the mesh to be unsubmitted. It could be used with defaultResource aswell. Since a mesh have its unique upload/default resource anyways